l10n_it_fattura_accompagnatoria/__manifest__.py

```diff
@@ -1,14 +1,15 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Lorenzo Battistini - Agile Business Group
+# Copyright 2020 Simone Vanin - Agile Business Group
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
 
 {
     "name": "Fattura Accompagnatoria",
```

Comment:
Puoi cambiare il nome in `ITA - ...` come da linee guida https://www.odoo-italia.org/documentazione/12.0/sviluppo/guidelines.html#titolo-del-modulo?